automobile/sumo-interface.md

```diff
@@ -30,18 +32,24 @@ field `controllerArgs` in order to customize the behavior of the interface:
+| --maxVehicles            | Specifies the maximum number of vehicles included in Webots                                                                          | By default, a maximum of 100 vehicles are included in Webots                                     |
+| --rotateWheels           | Updates the orientation of the wheels of the vehicles in Webots                                                                      | By default, the wheels orientation is not updated                                                |
+| --radius                 | Specifies the visibility radius of the vehicles in meters (only vehicles closer to the viewpoint than this radius will be displayed) | By default, the visibility radius is equal to -1 which means that all the vehicles are displayed |
+| --enableHeight           | Specifies if height information should be extracted from the edge name                                                               | By default, all the vehicles are in the plane defined by y = 0                                   |
+| --d or --directory       | Specifies the directory where are located the SUMO network files                                                                     | By default the files should be located close to the world in a directory called `worldName_net`  |
+| --p --port               | Specifies which port SUMO and the interface should use to communicate                                                                | By default, the port 8873 is used                                                                |
+| --s --seed               | Specifies the seed of the SUMO random number generator (0 for the '--random' option of SUMO)                                         | By default, a seed of 1 is used                                                                  |
+| --useSumoDisplay         | Specifies if the gui view of SUMO should be displayed in a Webots display (only working in gui mode, and the Supervisor must have a display named 'sumo display')                                         | By default, the gui view is not displayed  |
+| --sumoDisplayRefreshRate | Specifies the refresh rate of the SUMO display (no effect if the SUMO display is not activated)                                      | By default, a refresh rate of 1000ms is used                                                                  |
+| --sumoDisplayZoom        | Specifies the initial zoom of the SUMO display in Webots (100 means no scaling)                                                      | By default, no scaling is performed                                                                  |
+| --sumoDisplayResizeImage | Specifies if the image should be resized to fit the SUMO display size or not                                                         | By default, the image is not resized                                                                  |
```

Comment:
This is not clear how this interact with the display zoom described above...
Does the resize adjust the width / height aspect ratio? Or is it an automatic scaling that doesn't affect the aspect ratio?